internal/testdata/objectives.go

```diff
@@ -60,6 +61,10 @@ func genericDFO() directfund.Objective {
 
 func genericVFO() virtualfund.Objective {
```

Comment:
Some godoc here would be nice, since this constitutes quite a significant change to the side effects of this function. Something mentioning that a "linear" ledger network will be created?

Reply:
This is a little tricky because:

- (unless I'm mistaken) the changing side-effects aren't visible to the the caller
- this function is internal, and is exported via the struct `vfoCollection`, which does apply godoc to its `GenericVFO` member

Electing to leave as-is for now, but open to further conversation.